libcontainer/selinux/selinux.go

```diff
@@ -58,16 +57,30 @@ func getSelinuxMountPoint() string {
 	}
 	selinuxfs = ""
 
-	mounts, err := mount.GetMounts()
+	f, err := os.Open("/proc/self/mountinfo")
 	if err != nil {
 		return selinuxfs
 	}
-	for _, mount := range mounts {
-		if mount.Fstype == "selinuxfs" {
-			selinuxfs = mount.Mountpoint
-			break
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		txt := scanner.Text()
+		sepIdx := strings.Index(txt, " - ")
```

Comment:
@cyphar Thanks. Replicated the original [comment](https://github.com/docker/docker/blob/4a7bd7eaef0038b006cf23adface84081e5aab3b/pkg/mount/mountinfo_linux.go#L65).